modules/stanford_basic_page_types/stanford_basic_page_types.module

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @file
+ * stanford_basic_page_types.module
+ */
+
+use Drupal\node\NodeInterface;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * Implements hook_metatags_alter().
+ */
+function stanford_basic_page_types_metatags_alter(array &$metatags, array $context) {
+  if (!empty($context['entity']) && $context['entity'] instanceof NodeInterface && $context['entity']->bundle() == 'stanford_page') {
+    $entity = $context['entity'];
+    // If we have something in the page description field,
+    // we want that to take over the description metatag.
+    if (!empty($entity->get('su_page_description')->getString())) {
+      $metatags['description'] = "[node:su_page_description]";
```

Comment:
This could just be part of the normal config. it doesn't need to be altered here.